lib/src/Base/Func/HyperbolicAnisotropicEnumerateFunction.cxx

```diff
@@ -275,6 +276,24 @@ void HyperbolicAnisotropicEnumerateFunction::setUpperBound(const Indices & upper
   initialize();
 }
 
+/* The marginal enumerate function */
+EnumerateFunction HyperbolicAnisotropicEnumerateFunction::getMarginal(const Indices & indices) const
+{
+  const UnsignedInteger inputDimension = getDimension();
+  indices.check(inputDimension);
+  const UnsignedInteger activeDimension = indices.getSize();
+  Point weightMarginal(activeDimension);
+  for (UnsignedInteger i = 0; i < activeDimension; ++i)
+    weightMarginal[i] = weight_[indices[i]];
+  const HyperbolicAnisotropicEnumerateFunction enumerateFunctionMarginal(weightMarginal, q_);
+  return enumerateFunctionMarginal;
+}
+
+/* The marginal enumerate function */
+EnumerateFunction HyperbolicAnisotropicEnumerateFunction::getMarginal(const UnsignedInteger i) const
+{
+  return getMarginal(Indices({i}));
+}
```

Comment:
then these can be removed

Reply:
Fixed at d17f1462144022b009756e26e0d7de62313a3207

Reply:
Oups, it fails.

I think that this is not true. See e.g. [TruncatedDistribution.hxx#L150](https://github.com/openturns/openturns/blob/af13c83d6a2b70e0300cea830f6e8ff9af70236f/lib/src/Uncertainty/Distribution/openturns/TruncatedDistribution.hxx#L150)

Reply:
Ok, fixed.